include/psa/crypto_types.h

```diff
@@ -150,35 +149,21 @@ typedef uint32_t psa_key_lifetime_t;
  * actually affect persistent keys at different levels is outside the
  * scope of the PSA Cryptography specification.
  *
- * This specification defines the following values of persistence levels:
+ * The PSA Cryptography specification defines the following values of
```

Comment:
This is not really what defined by the specification anymore, rather the status of what values are supported or not by Mbed TLS implementation. Thus rephrase this?

Reply:
Well, the enumeration below covers all the values defined by the spec, it just groups together those that are not supported by Mbed TLS, so I don't think the sentence is incorrect as it is.

Reply:
I've kept this passage with minor tweaks because I think it makes sense to document the value range even if Mbed TLS doesn't implement all possible values.